vector/src/main/java/im/vector/app/features/voicebroadcast/recording/VoiceBroadcastRecorderQ.kt

```diff
@@ -103,6 +105,7 @@ class VoiceBroadcastRecorderQ(
 
     override fun startRecord(roomId: String) {
         super.startRecord(roomId)
+        playbackTracker.updateCurrentRecording(AudioMessagePlaybackTracker.RECORDING_ID, emptyList())
```

Comment:
You should call `playbackTracker.stopPlayback()` when stopping the recording (it will change the playback state to IDLE) or rename the existing method/add a new one.

Eventually also call stopPlayback on pause & updateCurrentRecording on resume

Reply:
I added the tracker on stop. I don't prefer to manage pause state for the moment.